templates/default.yaml

```diff
@@ -555,6 +555,10 @@ guestInstallPrefix: null
 # 🟢 Builtin default: false
 plain: null
 
+# Keep the mounts, in "plain" mode.
+# 🟢 Builtin default: false
+plainMounts: null
```

Comment:
I fear the explosion of `plainFoo`, `plainBar`, `plainFooBar` properties.

Can we rather introduce `guestAgent.enabled: false` that will just disable the guest agent without affecting mounts

Reply:
The ask was only to be able to do virtiofs mounts with vz, without the guestagent (which is doable with this)